keyboards/crkbd/keymaps/rpbaptist/keymap.c

```diff
@@ -0,0 +1,234 @@
+                                      //`--------------------------'  `--------------------------'
+  ),
+
+  [_NAV] = LAYOUT( \
+  //,-----------------------------------------------------.                    ,-----------------------------------------------------.
+      _______, KC_WH_U,   TBBCK, KC_MS_U,   TBFWD, KC_BTN2,                      KC_PGUP, KC_HOME,   KC_UP,  KC_END, XXXXXXX,  KC_DEL,\
+  //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
+       KC_TAB, KC_WH_D, KC_MS_L, KC_MS_D, KC_MS_R, KC_BTN1,                      KC_PGDN, KC_LEFT, KC_DOWN, KC_RGHT, XXXXXXX, XXXXXXX,\
+  //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
+      _______, XXXXXXX, KC_WH_L, XXXXXXX, KC_WH_R, XXXXXXX,                      XXXXXXX, KC_ACL0, KC_ACL1, KC_ACL2, XXXXXXX, _______,\
+  //|--------+--------+--------+--------+--------+--------+--------|  |--------+--------+--------+--------+--------+-------+---------|
+                                          _______, _______, _______,    _______, S_NAV, _______ \
+                                      //`--------------------------'  `--------------------------'
+  ),
+
+  [_UTIL] = LAYOUT( \
+  //,-----------------------------------------------------.                    ,-----------------------------------------------------.
+        RESET, XXXXXXX, KC_MSTP, KC_VOLU, KC_MNXT, XXXXXXX,                      COLEMAK,  GAMING, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,\
+  //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
+      XXXXXXX, XXXXXXX, KC_MPRV, KC_VOLD, KC_MPLY, XXXXXXX,                      XXXXXXX, RGB_MOD, RGB_SPI, RGB_HUI, RGB_SAI, RGB_VAI,\
+  //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
+      XXXXXXX, KC_SLEP, XXXXXXX, KC_MUTE, XXXXXXX, XXXXXXX,                      RGB_TOG,RGB_RMOD, RGB_SPD, RGB_HUD, RGB_SAD, RGB_VAD,\
+  //|--------+--------+--------+--------+--------+--------+--------|  |--------+--------+--------+--------+--------+--------+--------|
+                                          _______, _______, _______,    _______, _______, _______ \
+                                      //`--------------------------'  `--------------------------'
+  )
+};
+
+uint32_t layer_state_set_user(uint32_t state) {
+  switch (biton32(default_layer_state)) {
+    case _COLEMAKDHM:
+      state = update_tri_layer_state(state, _NUM, _NAV, _UTIL);
+      break;
+    case _GAMING:
+      state = update_tri_layer_state(state, _GAMING_EXT, _NAV, _UTIL);
+      break;
+  }
+  return state;
+}
+
+#ifdef OLED_DRIVER_ENABLE
+oled_rotation_t oled_init_user(oled_rotation_t rotation) {
+  if (is_master) {
+    return OLED_ROTATION_270;
+  } else {
+    return OLED_ROTATION_180;
```

Comment:
I think you mean: 
```suggestion
    return OLED_ROTATION_90;
```

Reply:
Well, I tried it. The result wasn't pretty.

Reply:
Ooooh, okay, you're displaying a logo, that's my bad. 

In that case, yeah, you want the 180.  :)